venues/auai.org/UAI/2017/python/superuser-init.py

```diff
@@ -10,11 +10,15 @@
 ## Import statements
 import argparse
 import csv
-import sys
 import openreview
 from uaidata import *
 from subprocess import call
 
+import sys, os
+def get_path(path): return os.path.join(os.path.dirname(__file__), path)
```

Comment:
I'm not refactoring get_path into utils because finding the path of the executing script (not the module file utils.py) might not be possible. See the answer here:

http://stackoverflow.com/questions/2632199/how-do-i-get-the-path-of-the-current-executed-file-in-python